spec/factories/admin_sets.rb

```diff
@@ -22,8 +22,4 @@
       with_permission_template false
     end
   end
-
-  trait :public do
```

Comment:
I removed this trait in favor of using `read_groups` explicitly in specs that need public Admin Sets, primarily because the presence of this trait could communicate that we expect public Admin Sets to be a common case. I don't think we do.